backend/notam_api/views.py

```
@@ -30,6 +30,26 @@ def fetch_notams_data(start_location, end_location):
 
     return notams
 
+def check_airports(request):
+    # Get start and end locations from request
+    start_location = request.GET.get('start', 'OKC')  # Default to 'OKC' if not provided
```

Why not fail validation if a start/end is not provided?

---

It does fail. I just took David's code from the get_notams function but if you don't put anything in the text field it will throw the same error.